crates/ruma-common/src/events/room/member.rs

```diff
@@ -187,6 +207,29 @@ impl RedactedRoomMemberEventContent {
     pub fn details(&self) -> MembershipDetails<'_> {
         MembershipDetails { avatar_url: None, displayname: None, membership: &self.membership }
     }
+
+    /// Helper function for membership change.
+    ///
+    /// Since redacted events don't have `unsigned.prev_content`, you have to pass the `.details()`
+    /// of the previous `m.room.member` event manually (if there is a previous `m.room.member`
+    /// event).
```

Comment:
I think we should have a similar note on the other methods about `prev_content` being erased if the event it was referring to is redacted.

Reply:
As discussed in the matrix room, we always have a `prev_content` if the current event is not redacted.

I'll create the possible redacted type in a separate PR.